src/LearnositySdk/Request/Init.php

```diff
@@ -2,6 +2,7 @@
 
 namespace LearnositySdk\Request;
 
+use \Exception;
```

Comment:
We don't need this one anymore, do we?

Reply:
I so rely on JS linter, that I never check those unused imports. Is it possible to do the same in PHP? I've googled a bit, and found sth, but not sure if this is good: http://cs.sensiolabs.org/

As far as I know we have nothing like it in other codebases :/

Reply:
I don't know of one, but I agree it's a gap.